README.md

```diff
@@ -1,5 +1,9 @@
 # Zeppelin Converter
 
+### Requirements
+
+- Python 3.\*
```

Comment:
You should mention that it's tested on python 3.6, but it may work on other versions.